integration-tests/test_collection.py

```diff
@@ -106,3 +111,101 @@ def test_output_file_already_exists(insights_client, tmp_path):
     cmd_result = insights_client.run(f"--output-file={output_file}", check=False)
     assert cmd_result.returncode == 1
     assert f"ERROR: File {output_file} already exists." in cmd_result.stderr
+
+
+@pytest.mark.usefixtures("register_subman")
+def test_cmd_timeout(insights_client):
+    """
+    Test cmd_timeout config option
+    Client executes all commands with timeout. If the command doesn't finish within
+    a specified amount of seconds, it is killed.
+    Test that setting a custom amount of seconds to cmd_timeout config option works.
+    """
+    cmd_output_message = "Executing: [['timeout', '-s', '9', '10'"
+
+    insights_client.register()
+
+    insights_client.config.core_collect = True
```

Comment:
Hi, we need not specifically set core_collect to true because for versions of the Insights Client RPM 3.1.0 and newer, core collection is enabled by default. So please remove this step.

Reply:
oh okay, understood, done